tests/foreman/cli/test_hammer.py

```diff
@@ -30,7 +30,7 @@
 
 HAMMER_COMMANDS = json.loads(DataFile.HAMMER_COMMANDS_JSON.read_text())
 
-pytestmark = [pytest.mark.tier1, pytest.mark.upgrade, pytest.mark.e2e]
+pytestmark = [pytest.mark.tier1, pytest.mark.upgrade]
```

Comment:
Yeah I was considering this a lot, basically I expect the hammer commands and options to change vividly within stream so it would require a lot of effort to keep these tests green. To me it makes more sense to check the options after we branch when they get stabilized

Reply:
Still, some of the tests can be kept, right? Like `test_positive_hammer_shell`

Reply:
Brought that one back